.travis.yml

```diff
@@ -1,24 +1,24 @@
+dist: trusty
 language: cpp
 script: cmake
 
-matrix:
-  include:
-    - env: CLANG_VERSION=3.7 CPP=11 LIBCXX=On
-      os: linux
-      addons: *clang37
+addons:
+  apt:
+    sources:
+      - kubuntu-backports
+      - ubuntu-toolchain-r-test
+    packages:
+      - gcc-5
+      - g++-5
+      - cmake
 
-# Attempt to get the right GCC/Clang versions
-before_script:
-  - sudo add-apt-repository ppa:kubuntu-ppa/backports -y
-  - sudo apt-get update -qq
-  - sudo apt-get install cmake
-  - git clone --depth=1 https://github.com/avr-llvm/clang.git tools/clang
-  - git clone --depth=1 https://github.com/avr-llvm/compiler-rt.git projects/compiler-rt
+env:
+  - CLANG_VERSION=3.7 CPP=11 LIBCXX=On
```

Comment:
I actually thought that `CLANG_VERSION` was used for compiling LLVM. One of the Travis errors was that the version of GCC was too old, so I assumed that we were bootstrapping with GCC.

However, since I removed the LLVM build in this repo, I guess that wouldn't matter either way!

Would you rather bootstrap with clang?